rho/scancommand.py

```diff
@@ -239,7 +239,8 @@ def run_ansible_with_vault(cmd_string, vault_pass, ssh_key_passphrase=None,
                 child.logfile = logfile
                 child.expect(pexpect.EOF)
 
-            child.wait()
+            if child.isalive():
```

Comment:
I'm guessing you got an error with the plain `child.wait()`, but this if statement still looks like a race condition to me, since the process could finish between the `isalive()` and the `wait()`. Fine for this PR, but should we just get rid of pexpect and use subprocess directly? (Assuming the bug is in pexpect.)